dev/com.ibm.ws.jdbc/src/com/ibm/ws/rsadapter/impl/WSRdbManagedConnectionImpl.java

```diff
@@ -3821,6 +3855,10 @@ public final void setAutoCommit(boolean value) throws SQLException
     {
         if (value != currentAutoCommit || helper.alwaysSetAutoCommit()) 
         {
+            if(useDriverIsoLvl && (value == false) )
+                throw new SQLException("DataSource configured with an isolation level of 'NONE (0)'. "
```

Comment:
This message needs to be translated.

Reply:
When you do write a translatable message, leave out the part about the connection not being enlisted in a global transaction.  What the user needs to know is that setAutoCommit(false) is disallowed with TRANSACTION_NONE regardless of other circumstances.